### PR TITLE
Special security announcement may

### PR DESCRIPTION
**Pull request by anjai94**
This PR will merge following security advisories into the security documentation site. 

- WSO2-2022-2177
- WSO2-2022-2182

### CODE REVIEW

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    <td>
+<ul>
+<li>WSO2 Identity Server 5.9.0, 5.10.0, 5.11.0, 6.0.0</li>
+<li>WSO2 Identity Server as Key Manager 5.9.0, 5.10.0</li>
+<li>WSO2 API Manager 3.0.0, 3.1.0, 3.2.0, 4.0.0, 4.1.0</li>
+<li>WSO2 Open Banking AM 2.0.0, 3.0.0</li>
+<li>WSO2 Open Banking IAM 2.0.0, 3.0.0</li>
+</ul>
+</td>
+    <td>Append the following configuration lines to the "&lt;PRODUCT-HOME&gt;/repository/conf/deployment.toml" file and restart the server:
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md:119]: intead of using paths like this:
"&lt;PRODUCT-HOME&gt;/repository/conf/deployment.toml"

Shall we use,
`<PRODUCT-HOME>/repository/conf/deployment.toml`

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+If you are an open-source user or using a product version that is EOL (End of License) :
+
+You may migrate to the latest version of the product if the latest version is not listed under the list of the affected products. Otherwise, you may apply the relevant fixes to the product based on the public fixes as given below:
+
+* [https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/216](https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/216)
+
+Or else you may follow the mitigation steps given below.
+
+<table>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md:104]: If we are to have a table, we have to change the the column size. You may use something like : https://github.com/wso2/docs-is/blob/master/en/docs/deploy/get-started/install.md?plain=1#L855

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    <td>
+    <ul>
+<li>WSO2 Identity Server 5.3.0, 5.4.0, 5.4.1, 5.5.0, 5.6.0, 5.7.0, 5.8.0</li>
+<li>WSO2 Identity Server as Key Manager 5.3.0, 5.5.0, 5.6.0, 5.7.0</li>
+<li>WSO2 Open Banking AM 1.3.0, 1.4.0, 1.5.0</li>
+<li>WSO2 Open Banking IAM 1.3.0, 1.4.0, 1.5.0</li>
+</ul>
+</td>
+<td>
+Add the following line as the first entry of the "&lt;ResourceAccessControl&gt;" tag of  "&lt;PRODUCT-HOME&gt;/repository/conf/identity/identity.xml" file and restart the server:
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md:141]: file path and tags, function name, etc

`<ResourceAccessControl>` tag of `<PRODUCT-HOME>/repository/conf/identity/identity.xml

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]
Suggested few changes

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+title: Security Advisory WSO2-2022-2182
+category: security-announcements
+published: May 10, 2023
```
[the remark this answers is not in the document]
Comment (anjai94) [reply en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md:4]: I'm planing to announce it in the mid of may that's why i added this date.

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+version: 1.0.0
+severity: High
+cvss: "CVSS Score:  8.3 (CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:L)"
+---
+
+# Security Advisory WSO2-2022-2182
+
+<p class="doc-info">Published: May 10, 2023</p>
```
[the remark this answers is not in the document]
Comment (anjai94) [reply en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md:12]: I'm planing to announce it in the mid of may that's why i added this date.

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+| WSO2 Open Banking AM Accelerator    |           3.0.0 |           Refer to WSO2AM-4.0.0 update level since this vulnerability is originating from the base product | N/A |
+| WSO2 Open Banking IAM               |           2.0.0 |          208 | 1664976509070 |
+| WSO2 Open Banking IAM Accelerator   |           3.0.0 |           Refer to WSO2IAM-5.11.0 update level since this vulnerability is originating from the base product | N/A |
+
+
+If you are an open-source user or using a product version that is EOL (End of License) :
+
+You may migrate to the latest version of the product if the latest version is not listed under the list of the affected products. Otherwise, you may apply the relevant fixes to the product based on the public fixes as given below:
+
+
```
[the remark this answers is not in the document]
Comment (anjai94) [reply en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md:75]: I believe better to have it.

**Pull request closed without merging**